pyup/bot.py

```diff
@@ -455,28 +455,17 @@ def create_pull_request(self, title, body, new_branch):
         # if we have a bot user that creates the PR, we might run into problems on private
         # repos because the bot has to be a collaborator. We try to submit the PR before checking
         # the permissions because that saves us API calls in most cases
+        config = dict(title=title,body=body, new_branch=new_branch,
+                      config=self.config)
         if self.bot_token:
             try:
-                return self.provider.create_pull_request(
-                    repo=self.bot_repo,
-                    title=title,
-                    body=body,
-                    base_branch=self.config.branch,
-                    new_branch=new_branch,
-                    pr_label=self.config.label_prs,
-                    assignees=self.config.assignees
-                )
+                return self.provider.create_pull_request(repo=self.bot_repo, **config)
             except NoPermissionError:
                 self.provider.get_pull_request_permissions(self.bot, self.user_repo)
 
         return self.provider.create_pull_request(
             repo=self.bot_repo if self.bot_token else self.user_repo,
-            title=title,
-            body=body,
-            base_branch=self.config.branch,
-            new_branch=new_branch,
-            pr_label=self.config.label_prs,
-            assignees=self.config.assignees
+            **config
```

Comment:
This is the only thing I'm a bit concerned about. The code for the online service is subclassing internally quite a lot and having functions with parameters helps quite a lot to fix errors.

Maybe pass provider specific code as `**extra`?

Reply:
Thanks for the feedback - I like the `**extra` idea.

Reply:
Will try to follow up as soon as possible.

Reply:
So basically you want to keep the list of keyword arguments and pass `extra=self.config` additionally? Or am I missing something?

Reply:
Exactly 👍

Reply:
Addressed your comments in  d34f29e and 34b70d2